src/Aurora.Infrastructure/DIExtensions.cs

```diff
@@ -1,13 +1,22 @@
 ﻿using Aurora.Application.Contracts;
+using Aurora.Infrastructure.Config;
+using Aurora.Infrastructure.Contracts;
+using Aurora.Infrastructure.Scrapers;
+using Aurora.Infrastructure.Services;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace Aurora.Infrastructure
 {
     public static class DIExtensions
     {
-        public static IServiceCollection AddInfrastructure(this IServiceCollection services)
+        public static IServiceCollection AddInfrastructure(this IServiceCollection services, IConfiguration config)
         {
             services.AddScoped<ISearchScraperCollector, SearchScraperCollector>();
+            services.AddScoped<PornhubScraper>();
+            services.AddScoped<IWebClientService, WebClientService>();
+            services.AddScoped<DriverInitializer>();
+            services.Configure<SeleniumConfig>(option => config.GetSection("Selenium").Bind(option));
```

Comment:
Why use bind?

Reply:
How would it convert section to config otherwise?

Reply:
but okay, it does the same, so no way to worry about it too much